packages/components/src/components/table/style.css

```diff
@@ -25,6 +25,13 @@
 	div.pagination {
 		order: 2;
 	}
+    
+	div.pagination kol-pagination  {
+		display: flex;
+		flex-wrap: wrap;
+		overflow-x: auto;
```

Comment:
So wird die Paginierung scrollbar, nehme ich an. Könnten wir sie stattdessen nicht in eine neue Zeile umbrechen lassen?